zebra/zebra_pbr.c

```diff
@@ -336,14 +336,16 @@ bool zebra_pbr_ipset_entry_hash_equal(const void *arg1, const void *arg2)
 	return true;
 }
 
-void zebra_pbr_iptable_free(void *arg)
+static void _zebra_pbr_iptable_free_ctx(void *arg, bool all)
 {
 	struct zebra_pbr_iptable *iptable;
 	struct listnode *node, *nnode;
 	char *name;
 
 	iptable = (struct zebra_pbr_iptable *)arg;
-	hook_call(zebra_pbr_iptable_update, 0, iptable);
+
+	if (all)
+		hook_call(zebra_pbr_iptable_update, 0, iptable);
```

Comment:
What? Why is this change being made?

Reply:
without 'all' keyword, when deleting context, the plugin context is freed. this is how I was testing it. and the feeling was that the plugin context was freed once configured. this was non sense for me.
with that, I can avoid flushing plugin context when it comes to handle local contexts created from zapi.

Reply:
@pguibert6WIND please take another look and determine if your nack has been addressed or not.

Reply:
If this fixes your use case then I'm fine with it. However, please add a comment near the function because it is entirely non-obvious what this does as there are *no* subscribers to the hook in the current code.